internal/graphql/resolver/meshsync.go

```diff
@@ -11,27 +13,74 @@ import (
 )
 
 var (
-	meshsyncYaml = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/samples/meshery_v1alpha1_meshsync.yaml"
+	meshsyncYaml              = "https://raw.githubusercontent.com/layer5io/meshery-operator/master/config/samples/meshery_v1alpha1_meshsync.yaml"
```

Comment:
@DelusionalOptimist, we want to stay away from manifests and stick with Helm charts, yes? Also, there is a MeshKit function that we should be reusing here, yeah?

Reply:
Yes @leecalcote, the upstream code isn't using this. Its just a left over which I missed while cleaning up other references to manifests.
Also, as far as I can tell, it isn't being used even in this PR, right @bariqhibat? In that case, I think we should remove this :+1:

Reply:
Yeah, this is unused. I'll delete this